sql/setup.sql

```diff
@@ -0,0 +1,11 @@
+ DROP TABLE if EXISTS booksTable;
+
+ CREATE TABLE booksTable (
+  id BIGINT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  img TEXT NOT NULL,
+  rating TEXT, 
+  title TEXT,
+  price INTEGER NOT NULL,
```

Comment:
The price is coming back as a string '$54.24' so...it's not an integer. Make sure to check your console log after you've scraped.